spec/models/send_to_report_service_job_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'spec_helper'
+
+describe "SendToReportServiceJob" do
+  before(:each) do
+    allow(Sequence).to receive(:find).and_return(sequence)
+    allow(ReportService::ResourceSender).to receive(:new).and_return(sender)
+    allow(ReportService).to receive(:configured?).and_return(report_service_configured)
+  end
+
+  let(:report_service_configured) { true }
+  let(:report_service_response) { { "status" => 200, "success" => true } }
+
+  let(:payload_hash) { "FACF4D8E-7AD9-489C-8CEC-58D004500795" }
+  let(:senderOpts) do
+    {
+      payload_hash: payload_hash,
+      send: report_service_response
+    }
+  end
+
+  let(:sender) { double("Sender", senderOpts) }
+
+  let(:sequence_update_time) {  1.minute.ago }
+  let(:last_report_service_hash) { "" }
+  let(:sequence) do
+    double("Sequence", {
+      type: 'Sequence',
+      id: 12,
+      updated_at: sequence_update_time,
+      last_report_service_hash: last_report_service_hash,
+      update_column: true
+    })
+  end
+
+  let(:queuetime) { Time.now }
+  let(:job) do
+    publishable_id = sequence.id
+    publishable_type = sequence.type
+    SendToReportServiceJob.new(publishable_type, publishable_id, queuetime)
+  end
+
+  it "the job should exist" do
+    expect(job).not_to be_nil
+  end
+
+  describe "#perform" do
+
+    describe "when the report service is not configured" do
+      let(:report_service_configured) { false }
+      it "should not raise an excpetion" do
+        expect {  job.perform }.not_to raise_error
+      end
+      it "should not send a request to the report server" do
+        expect(sender).not_to receive(:send)
+        job.perform
+      end
+    end
+
+    describe "when the published item has been modified afeter enqueing" do
+      let(:queuetime) { 1.hour.ago }
+      it "the job should not throw an exception" do
+        expect { job.perform }.not_to raise_error
+      end
+      it "the job should not make a requst to the report server" do
+        expect(sender).not_to receive(:send)
+        job.perform
+      end
+    end
+
+    describe "when the published item has been modified afeter enqueing" do
```

Comment:
This block: https://github.com/concord-consortium/lara/blob/9213d47c2d3ba64f1cbf69433ac7c8ebe3386b56/spec/models/send_to_report_service_job_spec.rb#L70-L79 

looks like a duplicate of the block above